packages/turf-angle/test.js

```diff
@@ -3,7 +3,7 @@ const path = require("path");
 const glob = require("glob");
 const load = require("load-json-file");
 const write = require("write-json-file");
-const sector = require("@turf/sector");
+const sector = require("@turf/sector").default;
```

Comment:
Do we know why this import out of all of them is the only one to need .default added?

Reply:
I think it has something to do with the other packages being written in typescript, which was built using `tsc` directly instead of using `rollup`. (We need .default when requiring because of https://tsup.egoist.dev/#interop-with-commonjs why this is needed)

I think I can fix this by only enabling cjsInterop if it's a typescript project, but that that migth complicate the setup. Let me know if this is something we want and I can try to get something working this weekend.